packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -616,40 +635,6 @@ contract DependencyRegistryV0 is
         );
     }
 
-    /**
-     * @notice Adds a new core contract to the list of supported core contracts.
-     * @param contractAddress Address of the core contract to be added.
-     */
-    function addSupportedCoreContract(address contractAddress) external {
-        _onlyAdminACL(this.addSupportedCoreContract.selector);
-        _onlyNonZeroAddress(contractAddress);
-
-        require(
-            // @dev the add function returns false if set already contains value
-            DependencyRegistryStorageLib.s().supportedCoreContracts.add(
-                contractAddress
-            ),
-            "Contract already supported"
-        );
-        emit SupportedCoreContractAdded(contractAddress);
-    }
-
-    /**
-     * @notice Removes a core contract from the list of supported core contracts.
-     * @param contractAddress Address of the core contract to be removed.
-     */
-    function removeSupportedCoreContract(address contractAddress) external {
-        _onlyAdminACL(this.removeSupportedCoreContract.selector);
-        require(
-            // @dev the remove function returns false if set does not contain value
-            DependencyRegistryStorageLib.s().supportedCoreContracts.remove(
-                contractAddress
-            ),
-            "Core contract already removed or not in set"
-        );
-        emit SupportedCoreContractRemoved(contractAddress);
-    }
-
```

Comment:
would be acceptable to leave this as commented code with a note that it was removed in an upgrade. up to you!

Reply:
added this back in with a code comment!